src/protocols/secure_channel/SessionEstablishmentExchangeDispatch.h

```diff
@@ -36,13 +36,16 @@ class SessionEstablishmentExchangeDispatch : public Messaging::ExchangeMessageDi
 
     virtual ~SessionEstablishmentExchangeDispatch() {}
 
-    CHIP_ERROR Init(TransportMgrBase * transportMgr)
+    CHIP_ERROR Init(Messaging::ReliableMessageMgr * reliableMessageMgr, TransportMgrBase * transportMgr)
```

Comment:
I guess this is OK for now, but as a followup: Why do we need to have a `ReliableMessageMgr` here at all?  I just checked, and all uses of `ExchangeMessageDispatch::mReliableMessageMgr` seem to be inside `ExchangeMessageDispatch::SendMessage` and in there it could get it from the `ReliableMessageContext`, no?  We'd need to make the getter public, but I think that should be ok....

Reply:
Yes, once we change the getter to public, we can remove it from this API.